sfa_dash/filters.py

```diff
@@ -77,10 +77,24 @@ def is_number(num):
     return str(num).isnumeric()
 
 
+def climate_zone_links(zones):
+    def reference_zone_link(zone_name):
+        if 'Reference Region' in zone_name:
+            zone_number = zone_name.split(' ')[-1]
+            return f'<a href="https://solarforecastarbiter.org/climatezones/#region{zone_number}">{zone_name}</a>'  # noqa: E501
+        else:
+            return zone_name
+    if not zones:
```

Comment:
If we can assume that `zones` is always a python list instead of numpy array then reverse the logic so this reads `if zones: ... else: return ["No Climate Zone"]`. If we cannot assume that, then the statement should be `if zones is not None:`